src/core/xfa/bind.js

```diff
@@ -545,6 +545,12 @@ class Binder {
           // to have something to match with the given expression.
           // See http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.364.2157&rep=rep1&type=pdf#page=199
           match = createDataNode(this.data, dataNode, ref);
+          if (!match) {
+            // For example if the node contains a .(...) then it isn't
+            // findable.
+            // TODO: remove this when .(...) is implemented.
```

Comment:
Can you file an issue to track this TODO (and attach a PDF that uses it)?

Reply:
https://github.com/mozilla/pdf.js/issues/13688